pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
   "numpy-quaternion",
   "pre-commit",
   "pytest",
+  "pytest-cov",
```

Comment:
Does not have to be decided in this PR, but do we want this in the default installation configuration? Or do we want a special "development" config?

Reply:
Good point - didn't think about that. For this PR I would leave it as is and we could adapt in within #152

Reply:
Sounds like a good plan!